modules/core/src/lib/tooltip.ts

```diff
@@ -81,8 +81,8 @@ export default class Tooltip {
       Object.assign(el.style, displayInfo.style);
     }
     this.isVisible = true;
-    el.style.display = 'block';
-    el.style.transform = `translate(${x}px, ${y}px)`;
+    el.style.display = el.style.display || 'block';
+    el.style.transform = el.style.transform || `translate(${x}px, ${y}px)`;
```

Comment:
I don't think this will work. `el.style.transform` will always be defined after the first time it is rendered.

You need to check `displayInfo.style` instead.

Reply:
I see. So, the tooltip element doesn't get removed after use. This will indeed use transform and display style from the last time it was set until it is changed by the user explicitly. But this is the behavior with all the style properties.

I made a codepen based on the iconlayer example to show this. https://codepen.io/nvn-nil/pen/eYPxMXP

We'll need to do `el.style = {}` or something in the `!displayInfo` block to prevent this from happening.

I'll use displayInfo for this check here. I can open another issue for the persisting styles or I can fix it here if you think that is not the intended behavior.

Reply:
I moved 
```js
    el.style.display = 'block';
    el.style.transform = `translate(${x}px, ${y}px)`;
```
before all the displayInfo blocks since it being at the end is what's overwriting the user set styles.